app/assets/javascripts/app/models/person.js

```diff
@@ -8,6 +8,10 @@ app.models.Person = Backbone.Model.extend({
   initialize: function() {
     if( this.get('profile') )
       this.profile = new app.models.Profile(this.get('profile'));
```

Comment:
Please also wrap that block in curly braces. I know that you didn't touch this part of the code but you added the new if block below and now it became even more confusing.